tests/playwright/src/model/pages/images-page.ts

```diff
@@ -36,16 +36,46 @@ export class ImagesPage extends MainPage {
   readonly loadImagesFromTarButton: Locator;
   readonly addArchiveButton: Locator;
   readonly confirmLoadImagesButton: Locator;
+  readonly deleteAllUnusedImagesCheckbox: Locator;
+  readonly bottomAdditionalActions: Locator;
```

Comment:
this locator is already in the parent class. No need to redefine it here, right?

Reply:
You're right, I did not see it in parent, fixed it now.